cozy-registry.example.yml

```diff
@@ -45,6 +45,7 @@ swift:
 # Conservation configuration allows to manage paramaters of how many app versions
 # of each channel must be kept during cleaning tasks
 conservation:
+  enabled: false # Specifies if the app cleaning task is enabled
```

Comment:
It is not clear to me if `conservation.enabled` means that the jobs for cleaning will be run, or not.

Reply:
And is this parameter also used for manual executions?

Reply:
> It is not clear to me if `conservation.enabled` means that the jobs for cleaning will be run, or not.

`conservation.enabled = false` means the job won't be executed. Maybe `conservation.remove_enabled` would be a better name ? 

> And is this parameter also used for manual executions?

This parameter is only intended to be used by background jobs, not CLI.

Reply:
Maybe `conservation.enable_background_cleaning`

Reply:
Yep, better :)